examples/python/gui_demo.py

```diff
@@ -329,7 +322,7 @@ def tree_add_component(self, parent_node_id, component):
                 component_name = 'Inputs/Outputs'
 
         self.tree.insert(parent_node_id, tk.END, iid=component_node_id,
-                         text=component_name, open=True, values=(component_node_id))
+                         text=component_name, open=True, values=(component_node_id,))
```

Comment:
Not a big deal, but could be removed

Reply:
I added it for a reason. In that case, Python leave values as tuple:

```python
>>> ("first second")
'first second'

>>> ("first second",)
('first second',)
``` 
In the first case, you have a problem with reading values. Ttk splits string.

```python
>>> item = self.tree.item(selected_item)
>>> node_unique_id = item['values'][0]
>>> node_unique_id
'first'
```

Reply:
Sorry. I was not clear. I mean extra comma